packages/expo-media-library/ios/EXMediaLibrary/EXMediaLibrary.m

```diff
@@ -401,25 +404,28 @@ - (NSDictionary *)constantsToExport
   }
   
   PHAsset *asset = [EXMediaLibrary _getAssetById:assetId];
+    
+  BOOL shouldDownloadFromNetwork = [[options objectForKey:EXMediaLibraryShouldDownloadFromNetworkKey] boolValue] ?: @YES;
```

Comment:
```suggestion
  BOOL shouldDownloadFromNetwork = [[options objectForKey:EXMediaLibraryShouldDownloadFromNetworkKey] boolValue] ?: YES;
```
From CI:
```
❌  /Users/runner/work/expo/expo/packages/expo-media-library/ios/EXMediaLibrary/EXMediaLibrary.m:408:114: incompatible operand types ('int' and 'NSNumber *')
```

Sorry for incorrect suggestion earlier :/

Reply:
No problemo 😄